docs/faqs-and-tips.md

```diff
@@ -60,3 +60,4 @@ Just disable your debugger in Xcode. 'Project name' -> 'Edit Scheme...' -> 'Run'
   - Try an isolate the problem with a template React Native Firebase app. This [bash script](https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh) is particularly helpful in making an empty template app.
 - Advice on supporting multiple environments (for example, dev, prod, maybe also staging, qa) for your React Native Firebase App: [#3504](https://github.com/invertase/react-native-firebase/issues/3504)
 - Using [Fastlane for iOS deployment](https://docs.fastlane.tools/getting-started/ios) together with [RN Firebase Crashlytics](https://rnfirebase.io/crashlytics/usage) within CI has been observed to cause builds that hang indefinitely. Using `setup_ci(force: true)` before building the application may solve the issue.: [#3706](https://github.com/invertase/react-native-firebase/issues/3706)
+- Be careful if you are using a VPN, on an Android emulator it might completely mess up the network adapter causing network calls to never return, not only on firebase but on the entire emulated phone, other firebase modules might also become unresponsive behind VPNs.
```

Comment:
```suggestion
- Be careful if you are using a VPN, Google blocks many VPN IPs causing "unavailable" errors for various firebase API calls, and on an Android emulator it might completely mess up the network adapter causing network calls to never return, not only on firebase but on the entire emulated phone.
```